README.md

```diff
@@ -124,10 +124,11 @@ yarn add @minify-html/node
 TypeScript definitions are available.
 
 ```ts
+import { Buffer } from "node:buffer";
```

Comment:
I don't think this is necessary, as it's a global by default. Although it provides more explicitness so it should be OK.